daisy_workflows/image_import/suse/suse_import/README.md

```diff
@@ -0,0 +1,26 @@
+This package supports translating OpenSUSE and SLES distributions to run on Google Compute Engine.
```

Comment:
There's a lot to digest in this large PR. Consider extending this README.md with a description on what each sub-module/package is responsible for so that, in the future, engineers can onboard faster when making changes in this module.

Reply:
That's good feedback :)  I updated the README to document the code's public interface, and describe its control flow for future maintainers.